techniques/systemSettings/misc/variableFromJsonFile/3.0/variableFromJsonFile.st

```diff
@@ -0,0 +1,178 @@
+#####################################################################################
+# Copyright 2019 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+bundle agent variable_from_json_file_RudderUniqueID
+{
+  vars:
+    &VARIABLE_FROM_JSON_FILE_JSON_FILE:{json_file |  "json_file[&i&]"        string => "&json_file&";
+}&
+    &TRACKINGKEY:{piuuid |  "trackingkey[&i&]"      string => "&piuuid&";
+}&
+    &VARIABLE_FROM_JSON_FILE_SOURCE:{source |  "sources[&i&]"      string => "&source&";
+}&
+    "complete_name" string => "&VARIABLE_FROM_JSON_FILE_VARIABLE_NAME&";
```

Comment:
indentation is 2 spaces for promiser type, 4 for classes condition, and 6 for promiser